openaps_reports.py

```diff
@@ -98,3 +105,12 @@ def recent_dose(self):
                 return self._read_json(Settings.SET_DOSE, [])
 
         return []
+
+    def data_age(self):
+        fn = self._get_report_path(Settings.PREDICT_GLUCOSE)
```

Comment:
Rather than choosing a specific file, can we just take the latest file? `all_filenames_and_data()` returns the files in last-modified order, and we're already calling it from the template. Let's move that call into the view and use the first item for the calculation